wled00/cfg.cpp

```diff
@@ -252,8 +253,16 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
           btnPin[s] = -1;
           pinManager.deallocatePin(pin,PinOwner::Button);
         }
+        //if touch pin, enable the touch interrupt on ESP32 S2 & S3
+        #ifdef SOC_TOUCH_VERSION_2    // ESP32 S2 and S3 have a fucntion to check touch state but need to attach an interrupt to do so
+          else if ((buttonType[s] == BTN_TYPE_TOUCH || buttonType[s] == BTN_TYPE_TOUCH_SWITCH)) 
+          {
+            touchAttachInterrupt(btnPin[s], touchButtonISR, touchThreshold<<4); //threshold on Touch V2 is much higher (1500 is a value given by Espressif example)
```

Comment:
I am talking about this. There is no counterpart in `set.cpp`.

Reply:
oh, guess I was wrong. will check.

Reply:
ok, I think I have it fixed now. I even added interrupt detach in the pin de-allocation function. touch now work immediately after saving button settings.